components/home/OurServices.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <section>
+    <v-container>
+      <app-section-descriptor
+        title="Services"
+        heading="Core Offerings"
+        description="Users care about Privacy and Security, set your app apart with the most developer friendly data storage and privacy platform. Takes minutes to integrate."
+        style="max-width: 64rem"
+      />
+      <div class="overview-features">
+        <v-stack align="center" sm-align="start" gap="20vw" sm-gap="1rem">
+          <div class="icon-container">
+            <v-image path="images/arcana-overview-auth.svg" alt="Arcana Auth" />
+          </div>
+          <div class="overview-feature">
+            <v-heading :level="2" class="overview-feature-title">
+              Arcana Auth
+            </v-heading>
+            <div class="overview-feature-description">
+              <v-text color="secondary" line-height="1.5" size="1.125rem">
```

Comment:
Please re-check the font sizes here against the design. The description text is reduced to 14px on mobile screen.